Makefile

```diff
@@ -53,6 +53,8 @@ $(GENERATED_FILES): %: %.in Makefile
 		-e 's|[@]bindir[@]|$(BINDIR)|g' \
 		-e 's|[@]ccagent[@]|$(TARGET)|g' \
 		"$<" > "$@"
+check:
+	./.ci/go-test.sh
```

Comment:
note to myself: This need to be a relative path to this makefile  so we call $(MKDIR)/.ci/go-test.sh